src/disbursements.ts

```diff
@@ -175,9 +175,9 @@ export default class Disbursements {
   public isPayerActive(
     id: string,
     type: PartyIdType = PartyIdType.MSISDN
-  ): Promise<string> {
+  ): Promise<{ result: boolean }> {
```

Comment:
Why not change the return type this simply boolean instead of the object with the boolean?

Reply:
Oh okay, I completely agree, will make the changes